node/core/av-store/src/lib.rs

```diff
@@ -87,6 +87,20 @@ pub struct Config {
 	pub path: PathBuf,
 }
 
+impl std::convert::TryFrom<sc_service::config::DatabaseConfig> for Config {
+	type Error = &'static str;
+
+	fn try_from(config: sc_service::config::DatabaseConfig) -> Result<Self, Self::Error> {
+		use sc_service::config::DatabaseConfig::{RocksDb, ParityDb, Custom};
+
+		match config {
+			RocksDb { path, cache_size } => Ok(Self { path, cache_size: Some(cache_size) }),
+			ParityDb { path } => Ok(Self { path, cache_size: None }),
```

Comment:
looking at the code of `av-store` it only works with rocksdb, but even then, I'm not sure it should be in the same path.
Otherwise it will try to override the column 0.

Reply:
Do we need an extra db for this? Can we not extend the db of the node with this one column?

Reply:
We could ofc use a different column in the same db, but the substrate column information [is private](https://github.com/paritytech/substrate/blob/1460807952ade12c4ba53e39e5928b87d89f545e/client/db/src/lib.rs#L321), we'd have to make sure not to overlap when a new column is added.

Reply:
Also the (substrate) cache_size info should not be used here (it will not be calculated properly anyway). Probably a default is fine.

Reply:
> @bkchr wdyt if we change the path to something like `path.parent().join("parachains").join("av_store")`?

Okay, if we go this way, I propose that `av_store` is a sub directory of the normal database directory to make commands like purge-chain work without changes.

Reply:
0720f5df